common/oidc-agent.robot

```diff
@@ -1,6 +1,6 @@
 *** Variables ***
 
-${oidc-agent.scope.default}   -s openid -s profile
+${oidc-agent.scope.default}   -s storage.read:/ -s storage.modify:/ -s storage.create:/ -s openid
```

Comment:
requesting `storage.create:/` here is redundant, according to the WLCG JWT profile